packages/client/hmi-client/src/services/workflow.ts

```diff
@@ -65,6 +65,28 @@ export class WorkflowWrapper {
 		const updatedNodeMap = new Map<string, WorkflowNode<any>>(updatedWF.nodes.map((n) => [n.id, n]));
 		const updatedEdgeMap = new Map<string, WorkflowEdge>(updatedWF.edges.map((e) => [e.id, e]));
 
+		if (pendingSave || isDragging) {
```

Comment:
I was not understanding why we would want to update the local version here 
Thank you for your help
May help to have a quick comment explaining this is to allow the user's outdated version to overwrite things prior to pulling